backend/src/routers/user.js

```diff
@@ -58,16 +58,22 @@ router.patch("/myFlex/api/v1/user/list", auth, async (req, res) => {
     try {
         const user = req.user
         const movieExists = await Movie.findOne({ id: req.body.id })
+        const formatResponse = Helper.filterWatchLaterMovie(req.body)
 
         if (movieExists === null) {
             const movie = await TMDBApi.movieDetails(req.body.id)
             const formatedMovie = Helper.formatMovie(movie)
             const movieID = new ObjectID()
             const mongoMovie = new Movie({ ...formatedMovie, _id: movieID })
             await mongoMovie.save()
-            user.movies = user.movies.concat({ _id: movieID, TMDB_Id: req.body.id })
+            user.movies = user.movies.concat({ _id: movieID, TMDB_Id: req.body.id, ...formatResponse })
         } else {
-            user.movies = user.movies.concat({ _id: movieExists._id, TMDB_Id: req.body.id })
+            if (Object.keys(formatResponse).length > 0) {
+                const movieIndex = user.movies.findIndex(movie => movie.TMDB_Id === req.body.id)
+                user.movies[movieIndex] = { ...user.movies[movieIndex]._doc, ...formatResponse }
+            } else {
+                user.movies = user.movies.concat({ _id: movieExists._id, TMDB_Id: req.body.id })
+            }
```

Comment:
Find movie in user List
If exists:
	change old user.list movie to new move with formatted response
	{…oldMovie, …formatedResponse}
else:
	does it exists in movie collection
	does not exist:
		add movie to movie colletion 

	{id: id, TMBD_ID,req.body.id, …formatedResponse}